builder/tencentcloud/cvm/step_run_instance.go

```diff
@@ -51,15 +52,18 @@ func (s *stepRunInstance) Run(ctx context.Context, state multistep.StateBag) mul
 	Say(state, "Trying to create a new instance", "")
 
 	// config RunInstances parameters
-	POSTPAID_BY_HOUR := "POSTPAID_BY_HOUR"
 	req := cvm.NewRunInstancesRequest()
 	if s.ZoneId != "" {
 		req.Placement = &cvm.Placement{
 			Zone: &s.ZoneId,
 		}
 	}
+	instance_charge_type := s.InstanceChargeType
```

Comment:
For local variables, Go tends to encourage camelCase in terms of naming, could you rename it to `instanceChargeType`?

Reply:
fixed